scripts/deploy.sh

```diff
@@ -7,17 +7,88 @@ pull_latest() {
 
 rebuild_regs() {
   # Build WCA regulations
-  # Use only one core because we get better backtraces without multiprocessing,
-  # and because we might be on the production server and should not eat up all
-  # cores just building the regulations.
-  regulations/make.py --num-workers 1 --wca
-  regs_folder=WcaOnRails/app/views/regulations
+  # Uses wrc, see here: https://github.com/cubing/wca-regulations-compiler
```

Comment:
This is really awesome work, and I think we should merge this up immediately. That said, this doesn't exactly fit my dream of the WCA website being "unaware" of how to build the regulations. IMO, there's a good amount of logic specific to the internals of the WRC here.

How do you feel about moving some or all this logic into the wca regulations compiler at some point?